test/cql-pytest/test-timestamp.py

```diff
@@ -86,7 +86,7 @@ def restrict_future_timestamp():
 # test will need to be replaced by a test for the new feature instead of
 # expecting an error message.
 def test_key_writetime(cql, table1):
-    with pytest.raises(InvalidRequest, match='PRIMARY KEY part k'):
+    with pytest.raises(InvalidRequest, match='WRITETIME is not legal on partition key component k'):
```

Comment:
If you change the error message check, this test will fail on Cassandra (please try `test/cql-pytest/run-cassandra test-timestamp.py::test_key_writetime`).
To allow this test to pass on both, you'll need to either check for two messages (e.g., `match='PRIMARY KEY part k|WRITETIME is not legal on partition key component k'` - it's a regex), or only check the common part (in this case it's only `' k'`... space and k...).

Reply:
Right.  I'll change to a regex using `|`.